### ISSUE TITLE
ldapadmin - upgrade to reCAPTCHA v2

### ISSUE THREAD

**Issue by fvanderbiest** [georchestra/georchestra#1001]
Today, we're receiving this kind of emails from google, asking we upgrade to v2

```
Dear Webmaster,

You're receiving this email because you're registered as a website administrator for a site that uses reCAPTCHA.

Google launched reCAPTCHA v2 (i.e. No CAPTCHA reCAPTCHA) in Dec, 2014. The new version provides a more simplified user experience than reCAPTCHA v1, with an enhanced spam fighting mechanism.

Action requested: please upgrade your site implementation to reCAPTCHA v2.
Please refer to the developer guide for integration details. We hope you enjoy the new reCAPTCHA.

Thank you,
reCAPTCHA Support
```

No timeframe is given.

**Comment by fvanderbiest** [comment]
Bumping for 15.12, hoping this will not break in between ...

**Comment by fvanderbiest** [comment]
Critical update on https://developers.google.com/recaptcha/docs/faq:

![capture du 2017-04-12 11-17-46](https://cloud.githubusercontent.com/assets/265319/24950551/b06f292e-1f71-11e7-9190-09c706deb2c4.png)

**Comment by fvanderbiest** [comment]
Action required: **reCAPTCHA v1 API shutdown on March 31, 2018**


Dear Webmaster,
--
You are receiving this email because you are registered as a website administrator using reCAPTCHA, and your website is still using reCAPTCHA v1, which will be turned off on March 31, 2018.
We announced the reCAPTCHA v1 deprecation in May 2016. Starting in November 2017, a small percentage of reCAPTCHA v1 traffic will begin to show a notice informing users that the old API will soon be retired. Any calls to the v1 API will not work after March 31, 2018.
To ensure continued functionality, you’ll need to update your website to a current version of reCAPTCHA. You can learn more about reCAPTCHA v2, Invisible reCAPTCHA and reCAPTCHA Android API in our Developer’s Guide. The new APIs are simple to implement and will streamline the captcha experience for your users. If you need help, you can engage in the reCAPTCHA Google Developer Group or post to Stack Overflow with the ‘recaptcha’ tag.
We hope that your upgrade will be seamless, and we’re confident you’ll be happy with the results.
Thank you,
reCAPTCHA Support

**Comment by fvanderbiest** [comment]
First proposal by Pierre Jego is https://github.com/georchestra/georchestra/pull/1928

**Comment by fvanderbiest** [comment]
Now blocking
![capture du 2018-04-04 15-02-25](https://user-images.githubusercontent.com/265319/38309067-40cc3c80-3819-11e8-9e26-11b5a4bec680.png)

**Comment by fvanderbiest** [comment]
Fixed on 15.12 with #1938:

![capture du 2018-04-17 13-47-41](https://user-images.githubusercontent.com/265319/38868035-0abb4bae-4246-11e8-9aaf-8a32150bfe19.png)
